tests/main/user-session-env/task.yaml

```diff
@@ -96,13 +87,6 @@ execute: |
                 MATCH "PATH=.*[:]?${SNAP_MOUNT_DIR}/bin[:]?.*" < "${user}-child-env"
                 # make sure that XDG_DATA_DIRS contains the default locations as well
                 MATCH 'XDG_DATA_DIRS=.*[:]?/usr/share[:]?.*' < "${user}-profile-env"
-                if ! os.query is-opensuse-tumbleweed; then
-                    # it was observed that on Tumbleweed XDG_DATA_DIRS seems to
-                    # have a default value which does not include
-                    # /usr/local/share, so apply the check only on remaining
-                    # systems
-                    MATCH 'XDG_DATA_DIRS=.*[:]?/usr/local/share[:]?.*' < "${user}-profile-env"
```

Comment:
Shouldn't this check stay, just be unconditional now? It was previously run everywhere except opensuse.

Reply:
Oops, of course! Good catch! I'll re-add it!

Reply:
And it turns out that the `if` is still needed :-)

I've updated the branch with a force-push, since leaving this commit in the history might have been misleading.